scripts/exgdas_atmos_verfrad.sh

```diff
@@ -110,7 +73,7 @@ if [[ -s ${radstat} && -s ${biascr} ]]; then
    satype_changes=0
    new_satype=$SATYPE
    for type in ${radstat_satype}; do
-      test=$(echo $SATYPE | grep $type | wc -l)
+      test=$(echo "$SATYPE" | grep "$type" | wc -l)
 
       if [[ $test -eq 0 ]]; then
```

Comment:
`test` is a bash command so we should probably avoid using it as a variable name
```suggestion
      type_count=$(echo "${SATYPE}" | grep "${type}" | wc -l)

      if (( type_count == 0 )); then
```